src/pykiso/lib/connectors/cc_proxy.py

```diff
@@ -60,28 +60,28 @@ def _cc_close(self) -> None:
         self.queue_out = Queue()
         log.debug("Close proxy channel")
 
-    def _cc_send(self, *args: tuple, **kwargs: dict) -> None:
+    def _cc_send(self, any_msg: Any, raw: bool = False) -> None:
```

Comment:
```suggestion
    def _cc_send(self, msg: Any, raw: bool = False) -> None:
```
--> Above layer use sometimes named parameter

Reply:
This will fail when using CAN and passing remote id

Reply:
yes, we will need to change the other auxiliaries accordingly. This is still ok because the user will not see it. it is an internal breaking change